src/tokenizer/stop_word_filter.rs

```diff
@@ -23,6 +23,26 @@ pub struct StopWordFilter {
     words: Arc<FxHashSet<String>>,
 }
 
+macro_rules! language {
+    ($lang:ident) => {
+        #[cfg(feature = "stopwords")]
+        /// Create a `StopWorldFilter` for the named language
+        pub fn $lang() -> Self {
+            Self::remove(
+                include_str!(concat!("stopwords/", stringify!($lang), ".txt"))
```

Comment:
It would be nicer to have it directly in a const &str array.
I assume you did not do that to make the source as clear as possible?

Reply:
> I assume you did not do that to make the source as clear as possible?

The main reason was not having any extra build-time processing step.

So I propose, I'll a small Python script which will download and process the Snowball files and produce a generated module that is checked into source control? (The data files themselves would not end up in source control then.)

Reply:
Did that and checked in both the script and its result.